workers/unity/Packages/io.improbable.gdk.tools/GdkToolsConfiguration.cs

```diff
@@ -22,6 +22,9 @@ public class GdkToolsConfiguration
         public string DevAuthTokenFilepath => Path.Combine(DevAuthTokenFullDir, "DevAuthToken.txt");
         public int DevAuthTokenLifetimeHours => TimeSpan.FromDays(DevAuthTokenLifetimeDays).Hours;
 
+        public string FullCodegenOutputPath => Path.GetFullPath(Path.Combine(Application.dataPath, "..", CodegenOutputDir));
```

Comment:
Can we make `CodegenOutputDir` private given we expect to use this path instead? Or nah?

Reply:
nah because we currently set it in toolsconfigwindow